### ISSUE TITLE
Export history to .xlsx file

### ISSUE THREAD

**Issue by Indranil012** [markusfisch/BinaryEye#394]
Please add an option to export history to .xlsx file or webhook.

**Comment by markusfisch** [comment]
Well, you can easily import CSV files into Excel already 😉

**Comment by Indranil012** [comment]
> Well, you can easily import CSV files into Excel already 😉

Yes but it takes sometime and for my application it only excepts .xlsx file.
And nother thing I need is custom header for the scanned code and I don't need the other information in exported file.
